config/kme_config.toml

```diff
@@ -38,17 +38,20 @@ ias_url = "https://api.trustedservices.intel.com/sgx/dev"
 ias_api_key = "112233445566778899aabbccddeeff00"
 
 # TEE enclave library to use.
-enclave_library = ""
-enclave_library_path = ""
+enclave_library = "libtcf-enclave.signed.so"
+enclave_library_path = "tc/sgx/trusted_worker_manager/enclave/build/lib/"
 
 num_of_enclaves = "1"
 # MRENCLAVE for WPE that will be allowed to register with this KME
-wpe_mrenclave = ""
+wpe_mrenclave = "CFF2E89D19A02886C653FD77B5D8314414D9C486155013CC0505671B894D053F"
```

Comment:
Let's not hard code this value here. One option is to read WPE MRENCLAVE value from a file. After WPE Enclave builds, use sgx tool to get the MRENCLAVE value and generate a file with that WPE MRENCLAVE. This file need not be checked in in git since this is a file generated during WPE build.
```suggestion
# This file is generated during WPE Enclave build
wpe_mrenclave_read_from_file = wpe_mr_enclave.txt
```

Reply:
Sounds good. Marked this as a todo with recommendation suggested.